Changelog.md

```diff
@@ -5,12 +5,35 @@ All notable Changes to the Julia package `Manopt.jl` will be documented in this
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.68] – August 2, 2024
+
+### Added
+
+* an Interior Point Newton Method, the `interior_point_newton`
+* a `conjugate_residual` Algorithm to solve a linear system on a tangent space.
+* `ArmijoLinesearch` now allows for additional `additional_decrease_condition` and `additional_increase_condition` keywords to add further conditions to accept additional conditions when to accept an decreasing or increase of the stepsize.
+* add a `DebugFeasibility` to have a debug print about feasibility of points in constrained optimisation employing the new `is_feasible` function
+* add a `InteriorPointCentralityCondition` check that can be added for step candidates within the line search of `interior_point_newton`
+* Add Several new functors
+  * the `LagrangianCost`, `LagrangianGradient`, `LagrangianHessian`, that based on a constrained objective allow to construct the hessian objective of its Lagrangian
+  * the `CondensedKKTVectorField` and its `CondensedKKTVectorFieldJacobian`, that are being used to solve a linear system within `interior_point_newton`
+  * the `KKTVectorField` as well as its `KKTVectorFieldJacobian` and ``KKTVectorFieldAdjointJacobian`
+  * the `KKTVectorFieldNormSq` and its `KKTVectorFieldNormSqGradient` used within the Armijo line search of `interior_point_newton`
+* New stopping criteria
+  * A `StopWhenRelativeResidualLess` for the `conjugate_residual`
+  * A `StopWhenKKTResidualLess` for the `interior_point_newton`
+
 ## [0.4.67] – July 25, 2024
 
 ### Added
 
 * `max_stepsize` methods for `Hyperrectangle`.
 
+### Added
+
+* Introduces a Riemannian Interior Point Newton Method based on the paper by Lai & Yoshise.
+* Add a `is_feasible` function and a detailed `DebugFeasibility` debug action
+
```

Comment:
Oh, and this I think shouldn't be in the "0.4.67" section.

Reply:
Ah, yes, I  must have missed that on a merge, maybe this merge went too well at that point ;)